src/mmw/js/src/modeling/models.js

```diff
@@ -270,16 +271,18 @@ var ScenarioModel = Backbone.Model.extend({
                     name: 'precipitation',
                     value: 1.0
                 }
-            ]
+            ],
+            // This is the md5 hash of the above default value
+            inputmod_hash: 'ab0d9b8a1d441b8ec7ddf81b5a49dd2d'
```

Comment:
Is it important to set this value initially?  I suppose that if we're defaulting a value, it's hard to pick up an event to calculate it on first run.  If we need it to be set, let's make it less brittle and just calculate the md5 via [hashlib(]https://docs.python.org/2/library/hashlib.html) so we can muck around with the default input with ease.